### DISCUSSION TITLE
i click on create new repo they show ma blanck page no any repo form for create repo  plz help me

### DISCUSSION THREAD

**Discussion by abubakarkhan12** [community/community#74908] [New to GitHub]
hello i am new on github first 2 to 3 days i create a repo but know hwen we click on create new repo they show ma blanck page no any repo form for create repo

**Comment by queenofcorgis** [comment]
![Screenshot 2023-11-28 at 9 50 01 AM](https://github.com/community/community/assets/92824518/f0cf416a-3047-4cd7-850c-664a3b55126b)


To confirm this form is not appearing when you click the button? If you could share a screenshot of what you are seeing that would be helpful for us to solve!
